### PR TITLE
Makes antitoxin and charcoal not the same thing

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
tweak: Antitoxin no longer purges chemicals
tweak: Charcoal heals less toxin damage the longer it is in your system
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
why did you
1. make anti-toxin even more generic
2. make charcoal shit at doing it's job

**Comment by Iamgoofball** [comment]
stop touching my codebabby while I'm trying to overhaul medbay

**Comment by MoreRobustThanYou** [comment]
@Iamgoofball 
Antitoxin heals toxin.
Charcoal gets rid of chemicals in your body.

**Comment by WJohn** [comment]
why have more chemicals, why not just compact them into having multiple functions so we don't have 500 chems with one single specific use that no one can ever remember how to make?

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 1]
```diff
@@ -394,7 +394,8 @@
 	taste_description = "ash"
 
 /datum/reagent/medicine/charcoal/on_mob_life(mob/living/M)
-	M.adjustToxLoss(-2*REM, 0)
+	if(prob(min(current_cycle-25, 10)))
```
Comment (swindly) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: This doesn't heal less the longer it's in the mob. The probability increases until 35 cycles and then stays at 10%.

**Comment by swindly** [comment]
Removing the purge from antitoxin means that there is no reagent that purges only toxins while there are three that purge all other reagents.

**Comment by ShadowDeath6** [comment]
tbh I think it's kind of dumb that a chemical can just purge toxins and not other medicine, it seems too easy

**Comment by XDTM** [comment]
It means it does nothing against overdoses, in turn charcoal purges everything, including other healing chems

**Comment by newfren** [comment]
Big fan of antitoxin not purging chems. Many situations where I don't want to remove a chem from someone while still needing to cure tox damage.

**Comment by MoreRobustThanYou** [comment]
i'd rather work on a revamp later

**Pull request closed without merging**